minibot/hardware/virtual_gpio/gpio.py

```diff
@@ -1,5 +1,5 @@
 """
-Minibot GPIO classes.
+Fake GPIO class to help with testing
```

Comment:
Nitpicky but maybe add comment about "Used when testing GPIO without a physical bot" (if that applies)?

Reply:
(also squash and merge pls)